nuget/lib/dependabot/nuget/nuget_config_credential_helpers.rb

```diff
@@ -27,32 +27,66 @@
         return if nuget_credentials.empty?
 
         File.rename(user_nuget_config_path, temporary_nuget_config_path)
+        File.write(
+          user_nuget_config_path,
+          <<~NUGET_XML
```

Comment:
I assume this is triggered because the variable name contains the word "password"? 
Technically this issue is pre-existing since "token" already contained the password and was used in the same way previously (stored as plain-text). The only difference here is I've renamed the variables so that they are more specific.

Any guidance on how to resolve this in Ruby? I'm not very familiar with the language and am unsure how to avoid this given the point of this code is to inject the plain-text password.